src/main/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+@Override
+ public void doWork() {
+    System.out.println("Truck started working")
```

Comment:
There is a missing semicolon at the end of this line. Java statements must end with a semicolon.